tools/lxdclient/client_network.go

```diff
@@ -49,13 +49,33 @@ type creator interface {
 	ProfileConfig(profile string) (*shared.ProfileConfig, error)
 }
 
+func checkBridgeConfig(client rawNetworkClient, bridge string) error {
+	n, err := client.NetworkGet(bridge)
+	if err != nil {
+		return err
+	}
+
+	if n.Config["ipv6.address"] != "none" {
+		return errors.Errorf(`juju doesn't support ipv6. Please disable LXD's IPV6:
```

Comment:
"Juju does not fully support IPv6-only LXD network config.
Please, disable the IPv6 network address by running:
$ lxc network set %s ipv6.address none
and re-bootstrap." ?